examples/webgpu_postprocessing_pixel.html

```diff
@@ -113,6 +113,7 @@
 			directionalLight.position.set( 100, 100, 100 );
 			directionalLight.castShadow = true;
 			directionalLight.shadow.mapSize.set( 2048, 2048 );
+			directionalLight.shadow.bias = - 0.0001;
```

Comment:
I don't know why but lights with `WebGPURenderer` seems to need a shadow bias more often. I suspect this is related to the shadow compare operation which is different compared to `WebGLRenderer`. The previous approach used `MeshDepthMaterial` with RGBA packing, the new approach uses a depth texture attachment. 

Increasing the depth texture type to `THREE.FloatType` did not help.